packages/components/src/external-link/index.jsx

```diff
@@ -1,88 +1,94 @@
 import { localizeUrl } from '@automattic/i18n-utils';
 import clsx from 'clsx';
 import { translate } from 'i18n-calypso';
-import { omit } from 'lodash';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
 import { ScreenReaderText, Gridicon } from '..';
 
 import './style.scss';
 
-class ExternalLink extends Component {
-	static defaultProps = {
-		iconSize: 18,
-		showIconFirst: false,
-		iconComponent: null,
-		localizeUrl: true,
-	};
+/**
```

Comment:
This is another exercise that worked more or less OK: generating those JSDoc with AI.  

Maybe we could convert to tsx as a follow-up. Not sure it makes sense to do it all in the same PR 🤷‍♂️